assets/js/index.js

```diff
@@ -0,0 +1,86 @@
+
+/* eslint-disable import/extensions */
+
+import player from './player.js';
+import gameBoard from './gameBoard.js';
+
+/* eslint-enable import/extensions */
+
+const response = (index, player, board, slots, status) => {
+  const response = board.modifyArray(index, player.val);
+  if (response) {
+    slots[index].innerHTML = player.val;
+    status = !status;
+  } else {
+    /* eslint-disable no-alert */
+    alert('Please choose another position');
+    /* eslint-enable no-alert */
```

Comment:
- [ ] Alert are considered bad practice because they take user's control from the application (and for many other reasons). Instead, you can use another solution, like a hidden container with a message, that becomes visible when you need to inform the user about something. Or you can try a fancy solution, like [alertify](https://alertifyjs.com/)